### ISSUE TITLE
plone.base references plone.app.layout - undeclared dependency indirection

### ISSUE THREAD

**Issue by jensens** [plone/Products.CMFPlone#3731]
`plone.base.interfaces.siteroot` imports `plone.app.layout.navigation.interfaces.INavigationRoot` and derives `IPloneSiteRoot` from it. This is an undeclared dependency indirection.

Todo: Move interface from plone.app.layout to plone.base and add a deprecated import to plone.app.layout.

See also https://github.com/plone/plone.base/pull/28

cc @gforcada

**Comment by gforcada** [comment]
Yes, thanks, I was about to create the PRs myself, but thanks for doing it 👍🏾
